lab-allie/test/track-routes-test.js

```diff
@@ -0,0 +1,373 @@
+'use strict';
+
+const server = require('../server.js');
+const chai = require('chai');
+const expect = chai.expect;
+const http = require('chai-http');
+// const Promise = require('bluebird');
+// const fs = Promise.promisifyAll(require('fs'), {suffix: 'Prom'});
+
+chai.use(http);
+
+describe('Track Route Tests', function() {
+  let app;
+  before(done => {
```

Comment:
how could you do a beforeEach and afterEach that created an instance, and cleaned the db after EACH test? new stretch :)